src/server/auth.ts

```diff
@@ -2,7 +2,7 @@ import { DrizzleAdapter } from "@auth/drizzle-adapter";
 import { type GetServerSidePropsContext } from "next";
 import NextAuth from "next-auth";
 import CredentialsProvider from "next-auth/providers/credentials";
-import bcrypt from "bcrypt";
+import bcrypt from "bcryptjs";
```

Comment:
do we remember if this was strictly necessary?

Reply:
the whole `fs` being unavailable on client side or whatever

Reply:
OH whoops yeah I should revert these changes